components/Voting/sessionPanel.styled.tsx

```diff
@@ -0,0 +1,84 @@
+/* tslint:disable:object-literal-sort-keys */
+/* tslint:disable:object-literal-key-quotes */
+import { breakpoint } from '../utils/styles/breakpoints'
+import { calcRem } from '../utils/styles/calcRem'
+import styled from '../utils/styles/theme'
+
+export const Panel = styled('li')(({ theme }) => ({
+  backgroundColor: theme.colors.lightGrey,
+  marginTop: '10px',
+  padding: '13px 15px',
+  listStyle: 'none',
+
+  '&:before': {
+    content: 'normal',
+  },
+
+  a: {
+    textDecoration: 'none',
+  },
+
+  ul: {
+    margin: 0,
+    padding: 0,
+  },
+}))
+
+export const Icon = styled('span')(({ theme }) => ({
+  fontSize: '18px',
+  float: 'right',
+  backgroundColor: theme.colors.primary,
+  color: '#fff',
+  borderRadius: '50%',
+  padding: '7px',
+  marginLeft: '10px',
+}))
+
+export const Title = styled('h4')(({ theme }) => ({
+  color: theme.colors.body,
+  textDecoration: 'none',
+  fontWeight: '700',
+  fontSize: '16px',
+  lineHeight: '1.5',
+  margin: '0 0 10px',
+}))
+
+export const Badge = styled('li')(() => ({
+  display: 'inline-block',
+  minWidth: '10px',
+  padding: '3px 7px',
+  fontSize: '12px',
+  fontWeight: '700',
+  lineHeight: '1',
+  marginRight: '5px',
+  color: '#f5f5f5',
+  textAlign: 'center',
+  whiteSpace: 'nowrap',
+  backgroundColor: '#666',
```

Comment:
We should have variables for the colours. It'll be theme.colours.SOMETHING. Whichever one covers this colour.